transactions/epoch/admin/reset_epoch.cdc

```diff
@@ -15,7 +15,6 @@ import FlowIDTableStaking from 0xIDENTITYTABLEADDRESS
 
 transaction(currentEpochCounter: UInt64,
```

Comment:
We should update these as well:
-  https://github.com/onflow/service-account/blob/main/templates/reset_epoch.cdc
- https://github.com/onflow/service-account/blob/main/templates/reset_epoch_with_end_staking_auction.cdc